charts/karpenter/templates/post-install-hook.yaml

```
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: helm-{{ .Release.Name }}-post-install-hook
```

is it customary to prefix with helm?
```suggestion
  name: {{ .Release.Name }}-post-install-hook
```